### PR TITLE
Remove unnecessary import comment from test file

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: config/configmapprovider/retrieved_test.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package configmapprovider // import "go.opentelemetry.io/collector/config/configmapprovider"
```
Comment (dmitryax) [inline config/configmapprovider/retrieved_test.go:15]: We add these annotations for usability automatically in all packages. Why removing here?

Comment (dmitryax) [reply config/configmapprovider/retrieved_test.go:15]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/Makefile#L74-L76

Comment (bogdandrutu) [reply config/configmapprovider/retrieved_test.go:15]: The tool adds only for non-test, check that almost all test don't have this, all other files have it.

Comment (dmitryax) [reply config/configmapprovider/retrieved_test.go:15]: oh, I see. tests are skipped... nvm then

**Review by dmitryax — APPROVED** [review]

**Pull request merged**